src/trix/config/block_attributes.coffee

```diff
@@ -5,6 +5,10 @@ Trix.config.blockAttributes = attributes =
   quote:
     tagName: "blockquote"
     nestable: true
+  heading:
```

Comment:
Let’s think through what will need to happen if we decide to add multiple header sizes in the future. Most likely those block attributes will be “header-1”, “header-2”, and so on, corresponding to \<h1>, \<h2>, etc.

Are we making things more difficult for ourselves by not going with “header-1” now? (I don’t mean to suggest that we are—I just want to pose the question.)